k8s/aditum/chart/aditum/templates/ingress.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: extensions/v1beta1
```

Comment:
Isn't `v1` available for Ingress? If so, please use it instead.

Reply:
We specifically changed it to v1beta1 because of errors we were getting with v1